internal/pkg/template/templates/app/cf.yml

```diff
@@ -1,16 +1,17 @@
 # Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: MIT-0
-AWSTemplateFormatVersion: '2010-09-09'{{$accounts := .Accounts}}{{$app := .App}}{{$services := .Services}}{{$svcTag := .ServiceTagKey}}
+AWSTemplateFormatVersion: '2010-09-09'{{$accounts := .Accounts}}{{$app := .App}}{{$workloads := .Workloads}}{{$svcTag := .ServiceTagKey}}
```

Comment:
wow i can't find it - where is .ServiceTagKey coming from?

Reply:
https://github.com/aws/copilot-cli/blob/5879fc0db12d0c13ce4d560b88ad03ca6db6f14d/internal/pkg/deploy/cloudformation/stack/app.go#L134

Reply:
And the reason why i keep the name is because the tag is literally "copilot-service" :coolguy-crying"